test/libsolidity/syntaxTests/multiSource/free_function_resolution_override_virtual.sol

```diff
@@ -16,4 +16,5 @@ contract D is C {
 // ----
 // Warning 2519: (s1.sol:65-134): This declaration shadows an existing declaration.
 // Warning 2519: (s2.sol:85-155): This declaration shadows an existing declaration.
+// Warning 2519: (s2.sol:85-155): This declaration shadows an existing declaration.
```

Comment:
I don't think this is really helpful.

Reply:
I guess it might be OK, because **solc** points at different shadowed declarations with different secondary locations. It is just that secondary locations are ignored by **isoltest**.

Reply:
Shouldn't we add all shadowed declarations to the list of secondaries instead?

Reply:
Yes. I did not realize that `SecondarySourceLocation` is a multi-location thing :).

I made a separate [commit](d9c49bd209e65a8cf01f4a9e81d82bf57c045f64)  to simplify review.